jobs/ci-kubernetes-e2e-gce-large-correctness.env

```diff
@@ -0,0 +1,28 @@
+### provider-env
+KUBERNETES_PROVIDER=gce
+PROJECT=kubernetes-scale
+KUBE_GCE_ZONE=us-east1-a
+CLOUDSDK_CORE_PRINT_UNHANDLED_TRACEBACKS=1
+
+### cluster-env
+NUM_NODES=2000
+MASTER_SIZE=n1-standard-32
```

Comment:
We can drop for n1-standard-32, but that's the highest that's automatically computed and anything above 500 nodes is given a master of n1-standard-32.
Ref: https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/config-common.sh#L33